### PR TITLE
Harvest rewards from StabilityPool, stake LQTY into LQTY Staking

### PR DESCRIPTION
**Pull request by kushal256**
Ready for review

### CODE REVIEW

**Review by lolchocotaco — COMMENTED** [review]
Overall looks good to me. love the comments. Aligned with policy as well.

File: contracts/allocators/LUSDAllocator.sol [hunk 1]
```diff
@@ -178,29 +201,83 @@ contract LUSDAllocator is Ownable {
...
+            return false;
+        }
+        // 1.  Harvest from LUSD StabilityPool to get ETH+LQTY rewards
+        lusdStabilityPool.withdrawFromSP(0);
+
+        // 2.  Stake LQTY rewards from #1.  This txn will also give out any outstanding ETH+LUSD rewards from prior staking
+        uint256 balanceLqty = IERC20(lqtyTokenAddress).balanceOf(address(this)); // LQTY balance received from stability pool
+        uint stakingEthRewards = 0;
+        uint stakingLUSDRewards = 0;
+        //TODO a mininum?
```
[the remark this answers is not in the document]
Comment (kushal256) [reply contracts/allocators/LUSDAllocator.sol:256]: Should I hardcode some minimum values?  For ETH maybe .25 ETH?  For LUSD maybe $1000?

[the remark this answers is not in the document]
Comment (lolchocotaco) [reply contracts/allocators/LUSDAllocator.sol:256]: Sync with policy on what a reasonable min is. Those values seem legit to me.

File: contracts/allocators/LUSDAllocator.sol [hunk 2]
```diff
@@ -178,29 +201,83 @@ contract LUSDAllocator is Ownable {
...
-        // TODO need to harvest LQTY rewards
+        uint256 stabilityPoolEthRewards = getETHRewards();
+        uint256 stabilityPoolLqtyRewards = getLQTYRewards();
+
+        //TODO a mininum?
+        if (stabilityPoolEthRewards == 0 && stabilityPoolLqtyRewards == 0) {
+            return false;
+        }
+        // 1.  Harvest from LUSD StabilityPool to get ETH+LQTY rewards
+        lusdStabilityPool.withdrawFromSP(0);
```
[the remark this answers is not in the document]
Comment (kushal256) [reply contracts/allocators/LUSDAllocator.sol:250]: From this conversation:  https://discord.com/channels/700620821198143498/818895484956835912/908031137010581594

Will add that link as a comment in the code, also

**Review by lolchocotaco — APPROVED** [review]
Looks good to me. going to merge this into allocators!

**Pull request merged**